### PR TITLE
Convert unsupported characters

### PR DESCRIPTION
**Pull request by enyia21**

### CODE REVIEW

File: src/main/java/formflow/library/pdf/PDFBoxFieldFiller.java [hunk 1]
```diff
@@ -70,9 +74,42 @@ private void setPdfField(String field, PDField pdField)
...
+
+      for(int i = 0; i < field.length(); i++){
+        Character character = field.charAt(i);
+        if(fontsIncludeGlyph(currentFonts, character)){
+          builder.append(character);
+        }else{
+          builder.append('?');
+        }
+
+        System.out.println("test");
```
Comment (coltborg) [inline src/main/java/formflow/library/pdf/PDFBoxFieldFiller.java:101]: Do you still need to print out `test`?
```suggestion
```

**Review by coltborg — CHANGES_REQUESTED** [review]
Tested and thing work as I expected. Left a comment to remove a `println`.

2023-11-27 at 16 40 13@2x

**Review by coltborg — APPROVED** [review]

**Comment by bseeger** [comment]
Is this one superseded by the one about OpenPDF?

**Comment by coltborg** [comment]
@enyia21 I think we can close this PR now?

**Comment by spokenbird** [comment]
Closing as we won't be usign this approach.

**Pull request closed without merging**